dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -302,15 +304,44 @@ public <T> T runChildTask(Task<T> task) throws MetadataDumperUsageException {
 
             if (requiredTasksNotSucceeded > 0) {
                 System.out.println(
-                        "* ERROR: " + requiredTasksNotSucceeded + " required task[s] failed.\n"
-                        + "* Output, including debugging information, has been saved to " + outputFile + "\n"
+                    "* ERROR: " + requiredTasksNotSucceeded + " required task[s] failed.\n"
+                        + "* Output, including debugging information, has been saved to "
+                        + outputFile + "\n"
                         + "**********************************************************"
                 );
-                if (exitOnError)
+                if (exitOnError) {
                     System.exit(1);
+                }
             }
 
+            logStatusSummary(state);
+            System.out.println("**********************************************************");
+        }
+    }
+
+    private void logStatusSummary(TaskSetState.Impl state) {
+        List<TaskState> taskStates = state.getTaskResultMap().values().stream()
+            .map(TaskResult::getState).collect(Collectors.toList());
+
+        Function<TaskState, Long> countState = desiredState -> taskStates.stream()
+            .filter(taskState -> taskState.equals(desiredState)).count();
+
+        long numberOfSuccess = countState.apply(TaskState.SUCCEEDED);
+        System.out.println("* SUCCESS: " + numberOfSuccess);
+
+        if (numberOfSuccess == taskStates.size()) {
+            return;
```

Comment:
Consider simplifying it by logging all the map keys with non-zero values. Additionally, I think there is no harm in logging enum name (e.g. `SUCCEEDED`) instead of `SUCCESS`:

```
taskStateCountMap.entrySet().stream()
        .filter(entry -> entry.getValue() != 0)
        .forEach(
            entry -> System.out.printf("* %s: %s%n", entry.getKey(), entry.getValue()));
```